package.json

```diff
@@ -42,6 +42,7 @@
     "url": "git://github.com/ampersandjs/ampersand-form-view"
   },
   "scripts": {
+    "validate": "jshint .",
```

Comment:
I made a PR to add precommit-hook in #31 which should negate the need for this line since it will automatically run jshint.